scenariorun/src/main/openapi/scenariorun.yaml

```diff
@@ -6,17 +6,37 @@ info:
   version: 0.0.8-SNAPSHOT
 
 servers:
-- url: 'https://api.azure.cosmo-platform.com'
-- url: 'http://localhost:4010'
+  - url: 'https://api.azure.cosmo-platform.com'
+  - url: 'http://localhost:4010'
 
 security:
-- oAuth2AuthCode: [ ]
+  - oAuth2AuthCode: [ ]
 
 tags:
-- name: scenariorun
-  description: ScenarioRun Management
+  - name: scenariorun
+    description: ScenarioRun Management
 
 paths:
+  /organizations/{organization_id}/scenarioruns/deleteall:
```

Comment:
Rename deleteall to deletehistoricaldata since it is a public exposition meaning must be clear